cmd/flags.go

```diff
@@ -128,7 +128,7 @@ var encS3Flag = cli.StringSliceFlag{
 
 var checksumFlag = cli.StringFlag{
 	Name:  "checksum",
-	Usage: "Add checksum to uploaded object. Values: MD5 (not stored), CRC32, CRC32C, SHA1 or SHA256. Requires server trailing headers (AWS, MinIO)",
+	Usage: "Add checksum to uploaded object. Values: CRC64NVME, CRC32, CRC32C, SHA1 or SHA256. Requires server trailing headers (AWS, MinIO)",
```

Comment:
But I just fixed this line 🥲 all good might be better to just push everyone to use the newer checksums.

Reply:
Yeah, just removing it from there is simpler.